overlord/devicestate/export_test.go

```diff
@@ -28,6 +28,9 @@ import (
 
 func MockKeyLength(n int) (restore func()) {
 	oldKeyLength := keyLength
+	if n < 1024 {
+		n = 1024
```

Comment:
That's misleading. If we can't take < 1024, let's panic instead and force the test to read correctly.